sale_delivery_rate/models/delivery_carrier.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import api, fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = 'delivery.carrier'
```

Comment:
My linter seems to be happy either way, and I like the tighter spacing a bit more. AFAIK, PEP 8 only requires space around class and method blocks, not inside them.